CMakeLists.txt

```diff
@@ -270,15 +270,12 @@ set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
 FetchContent_Populate(${extension_name}
   SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
   GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/slicersalt/${extension_name}
-  GIT_TAG        59dfda341c5a78f2511e6d0a66bdb09ed867604b # slicersalt-2020-01-15-d8c9fc0
+  GIT_TAG        9eb71073e4e782ab9a44f3ba708793735ac0c9c4 # slicersalt-2020-07-22-9eb7107
   GIT_PROGRESS   1
   QUIET
   )
 list(APPEND Slicer_EXTENSION_SOURCE_DIRS ${${extension_name}_SOURCE_DIR})
 
-list(APPEND Slicer_ADDITIONAL_DEPENDENCIES
-  python-ShapeVariationAnalyzer-requirements
```

Comment:
We should keep this line, this will ensure ShapeVariationAnalyzer requirements will be installed

Reply:
The different is that the `External_python-ShapeVariationAnalyzer-requirements.cmake` file is now provided by the bundled extension and not by SlicerSALT explicitly.

Reply:
Ah, I assumed that would be handled in the extension itself now. Looks like its still building but I'll push a fix.

Reply:
Humm ... you are right, the following is displayed:

```
-- SuperBuild -   python-ShapeVariationAnalyzer-requirements => Requires python[INCLUDED], python-pip[INCLUDED], python-setuptools[INCLUDED], python-wheel[INCLUDED], python-numpy[INCLUDED], python-scipy, 
-- SuperBuild -     python-scipy => Requires python[INCLUDED], python-setuptools[INCLUDED], python-numpy[INCLUDED], python-pip[INCLUDED], 
-- SuperBuild -     python-scipy[OK]
-- SuperBuild -   python-ShapeVariationAnalyzer-requirements[OK]
```

Let me check something

Reply:
Yes :) It is done for you. No need to change anything:

https://github.com/Slicer/Slicer/blob/24d715ee0b42b8461b1b4b3c8791ef5862a647cf/SuperBuild.cmake#L394-L400

Reply:
Documentation is here: https://github.com/Slicer/Slicer/blob/24d715ee0b42b8461b1b4b3c8791ef5862a647cf/CMakeLists.txt#L1005-L1029